src/OpenLoco/Vehicles/Vehicle.h

```diff
@@ -321,10 +321,23 @@ namespace OpenLoco::Vehicles
         void sub_4ADB47(bool unk);
         VehicleStatus getStatusTravelling() const;
         void getSecondStatus(VehicleStatus& vehStatus) const;
-        void sub_4BA7C7(uint8_t cargoType, uint16_t cargoQty, uint16_t cargoDist, uint8_t cargoAge, currency32_t profit);
+        void updateLastIncomeStats(uint8_t cargoType, uint16_t cargoQty, uint16_t cargoDist, uint8_t cargoAge, currency32_t profit);
     };
     static_assert(sizeof(VehicleHead) == 0x7A); // Can't use offset_of change this to last field if more found
 
+    struct LastIncome
```

Comment:
CargoIncome? `Last` shouldn't be part of the type name imo.

Reply:
I've changed it to `IncomeStats` I'm not sure you should really have cargo in the name but not really decided.